.github/actions/check-mdanalysis-install/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: Install and check
+description: Install MDAnalysis and check it's installed
+inputs:
+  install-version:
+    description: Input version to install
+    required: true
+    default: latest
+  include-tests:
+    description: Whether to install MDAnalysisTests
+    required: true
+    default: false
+  installer:
+    description: pip, conda, or mamba
+    required: true
+    default: pip
+  shell:
+    description: shell to use
+    required: true
+    default: bash
+outputs:
+  installed-version:
+    description: Installed MDAnalysis version
+    value: "${{ steps.print-version.outputs.installed-version }}"
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: run
+      uses: ./
+      with:
+        version: ${{ inputs.install-version }}
+        install-tests: ${{ inputs.include-tests }}
+        installer: ${{ inputs.installer }}
+        shell: ${{ inputs.shell }}
+    
+    - name: Check version
+      shell: python
+      run:
+        import MDAnalysis
+        assert MDAnalysis.__version__ == ${{ inputs.expected-version }}
+    
+    - name: Check MDAnalysisTests
+      if: ${{ matrix.include-tests }}
+      shell: python
+      run:
+        from MDAnalysis.tests.datafiles import PDB
```

Comment:
Why do this not also assert the expected version?

Reply:
does it have one? 

```
>>> import MDAnalysis.tests
>>> MDAnalysis.tests.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'MDAnalysis.tests' has no attribute '__version__'
```

Reply:
I was of the understanding that installing tests at all would fail if the versions didn't match, so assumed its presence meant it worked.

Reply:
Test MDAnalysisTests.__version__ directly?

It's extremely unlikely that you'd get MDATests of the wrong version number, but imho it flows better in checking the install that way (i.e. version of one and version of the other package)

Reply:
```
>>> import MDAnalysisTests
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'MDAnalysisTests'
```

It's not importable, I don't think. I could conda/pip list it and grep, but that's a lot of effort IMO. If you write the code for me I'll add it :p

Reply:
I was going to say, if you can't import MDAnalysisTests then that's a test failure in itself right?